client/src/components/mobile/ResourcesBannerMobile.js

```diff
@@ -1,46 +1,46 @@
-import React from 'react';

```

Comment:
uh so i deleted this file and added it back. i didn't make any changes in this file lol